### ISSUE TITLE
freqai - how to call labels from feature_engineering_expand_all to populate_indicators?

### ISSUE THREAD

**Issue by mhgutier** [freqtrade/freqtrade#8232]
## Describe your environment

  * Operating system: MAC OS 13.2.1
  * Python Version: 3.8.13(`python -V`)
  * CCXT version: 1.93.60(`pip freeze | grep ccxt`)
  * Freqtrade Version: freqtrade docker-2023.3.dev-305eda74 (`freqtrade -V` or `docker compose run --rm freqtrade -V` for Freqtrade running in docker)
  
## Your question

Hi,
how can i call informative_timeframe/pair from populate_indicators?
i am using FreqaiExampleHybridStrategy as my baseline and modifying it.

feature_engineering_expand_all
```
        dataframe["ema-period"] = ta.EMA(dataframe, timeperiod=period)
        dataframe["%-ema-period"] = dataframe["ema-period"]

```
populate_indicators
```
        dataframe.loc[
            (
            (dataframe[f'ema-period_20_{pair}_1h'] > dataframe[f"ema-period_50_{pair}_1h"]) &
            (dataframe[f'ema-period_50_{pair}_1h'] > dataframe[f"ema-period_100_{pair}_1h"])
            ) ,'AOTS']=1

```

basically i want to pass the label created from feature_engineering_expand_all to populate indicators.

in my config i have 
"indicator_periods_candles": [20,50,100]

im getting error : 
Unable to analyze candle (OHLCV) data for pair 1INCH/USDT:USDT: name 'pair' is not defined
also, i only want to pass the current pair.

**Comment by xmatthias** [comment]
well `name 'pair' is not defined` says it all (you're not even getting as far as "column is not available").

the correct way to access the pair in `populate_indicators()` is `metadata['pair']`.

**Comment by robcaulk** [comment]
You will also need to prepend the feature with `%%` if you wish for it to return to populate_indicators. Otherwise, FreqAI filters all features with a single `%`.
